resources/user-agents/browsers/blackberry/blackberry-7-1.json

```diff
@@ -28,108 +28,52 @@
       },
       "children": [
         {
-          "match": "Blackberry9800/#MAJORVER#.#MINORVER#*",
-          "device": "Blackberry 9800"
-        },
-        {
-          "match": "Mozilla/5.0 (BlackBerry; ?; BlackBerry 9105*applewebkit* (*khtml*like*gecko*) Version/#MAJORVER#.#MINORVER#*Safari/*",
```

Comment:
These are the "incomplete" patterns I was talking about.  They're duplicates of patterns below, they just seem to be lacking a few characters between the device and `*applewebkit`. After running browscap against the useragents I found, and looking at the coverage report, these were never hit (the longer pattern below was always used). Since there were no tests for these patterns, and I couldn't find any UAs that hit them, I figured it was safe to remove.